tools/utils/Utils/CommandLine/CommandLineException.cs

```diff
@@ -0,0 +1,43 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="CommandLineException.cs" company="Microsoft">
+//     Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+// Please use CLIApplication instead, this class is being deprecated.

```

Comment:
>CLIApplication instead, this class is being deprecated. [](start = 14, length = 55)

is anyone still using this? If no callers, delete.

Reply:
Yes, this is used, but not CommandLineUtils. I'll delete that and the comment here.